templates/content/basic/paket.dependencies

```diff
@@ -1,23 +1,14 @@
-storage: packages
-source https://nuget.org/api/v2
+source https://api.nuget.org/v3/index.json
 
-frameworks: net45, netcoreapp3.1, netstandard2.0
+nuget FSharp.Core >= 4.5.2
```

Comment:
I think this should pin to FSHarp.Core 4.7.2 exactly.

TPDTC must load into, for example, VS2019 devenv.exe and fsc.exe.  These use an FSharp.Core that is 4.7.2.  We do not want to encourage people to randomly move to higher FSharp.Core dependencies for their TPDTC since if they do their component will not be loadable into VS2019, and will randomly become unloadable in F# tooling each time they `paket update`.

Like all design time plugins, TPDTC should in general use the *lowest* FSHarp.Core and *lowest* framework dependency they can, in order to load into as many contexts as possible.

Reply:
> since if they do their component will not be loadable into VS2019

Not sure if I understand this point, but FSharp.Core is so complex that I am happy to admit that I may be wrong here. They would be loadable into any up-to-date VS 2019. Updating a type provider to .NET Standard 2.0 will make it not loadable in VS 2017. Updating that same type provider to depend on FSharp.Core 4.7.2 will only make it loadable in VS 16.7 or greater. What I specified will continue to have existing type providers work exactly where they work before.

I'm not married to `>= 4.5.2` though

Reply:
4.7.2 is fine, given VSMac and Mono are being updated